src/main/java/mil/dds/anet/beans/Report.java

```diff
@@ -37,6 +37,7 @@
 	DateTime releasedAt;
 	
 	DateTime engagementDate;
+	Integer engagementDayOfWeek;
```

Comment:
I prefer to make new fields `final` (at some point [e.g. when we have decided on a Checkstyle profile] we should consider changing them all).

Reply:
done